client/src/components/Individuals/IndividualsForm.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import FormRow from "../FormRow";
+import fetchData from "../../utils";
+
+const IndividualsForm = ({ setIndividualsOfSpecies }) => {
+  const { id } = useParams();
+
+  const [individualInfo, setIndividualInfo] = useState({
+    nickname: "",
+    scientist_email: "",
+    scientist_name: "",
+  });
+
+  const handleInput = (e) => {
+    setIndividualInfo((prevInfo) => {
+      return { ...prevInfo, [e.target.name]: e.target.value };
+    });
+  };
+
+  const addIndividual = async (e, speciesId) => {
+    e.preventDefault();
+    try {
+      const response = await fetchData(
+        `/species/${speciesId}/individuals`,
+        "POST",
+        {},
+        individualInfo
+      );
+
+      if (response) {
+        const {
+          data: { newIndividual },
+        } = response;
+        setIndividualsOfSpecies((prevIndividual) => [
+          ...prevIndividual,
+          newIndividual,
+        ]);
+      }
+    } catch (error) {
+      console.error("Error occured while creating event", error);
+    }
+    setIndividualInfo({
+      nickname: "",
+      scientist_email: "",
+      scientist_name: "",
+    });
+  };
+
+  console.log(individualInfo);
```

Comment:
Psst! You forgot this.

Reply:
Thank you for catching this. Will remove the unnessary console.logs!